src/bundle/Resources/views/themes/admin/account/login/index.html.twig

```diff
@@ -1,24 +1,56 @@
 {% extends '@ezdesign/account/base.html.twig' %}
 
 {%- block content -%}
+    <h2 class="ez-login__actions-headline">{{ 'base.sign_in'|trans|desc('Sign in to IBEXA DXP') }}</h2>
+    <p class="ez-login__actions-subheadline">{{ 'base.enter_credentials'|trans|desc('Enter your login credentials below.') }}</p>
+
     <form action="{{ path( 'login_check' ) }}" method="post" role="form">
         <fieldset>
             {% if error %}
-                <span>{{ error.message|trans }}</span>
+                <span class="ez-login__errors-container">{{ error.message|trans }}</span>
             {% endif %}
             <div class="form-group{% if error %} has-error{% endif %}">
-                <label class="ez-login__form-label" for="username">{{ 'authentication.username'|trans|desc('Username') }}:</label>
-                <input type="text" id="username" class="ez-login__form-input form-control" name="_username" value="{{ last_username }}" required="required" autofocus="autofocus" autocomplete="on"/>
+                <div class="ez-login__input-label-wrapper">
+                    <label class="ez-login__input-label" for="username">{{ 'authentication.username'|trans|desc('Username') }}</label>
+                </div>
+                <input 
+                    type="text" 
+                    id="username" 
+                    class="form-control ez-login__input ez-login__input--username" 
+                    name="_username" 
+                    value="{{ last_username }}" 
+                    required="required" 
+                    autofocus="autofocus" 
+                    autocomplete="on"
+                />
             </div>
             <div class="form-group{% if error %} has-error{% endif %}">
-                <label class="ez-login__form-label" for="password">{{ 'authentication.password'|trans|desc('Password') }}:</label>
-                <input type="password" id="password" class="ez-login__form-input form-control" name="_password" required="required"/>
+                <div class="ez-login__input-label-wrapper">
+                    <label class="ez-login__input-label" for="password">{{ 'authentication.password'|trans|desc('Password') }}</label>
+                    <a href="{{ path('ezplatform.user.forgot_password') }}">{{ 'authentication.forgot_password'|trans|desc('Forgot password?') }}</a>
+                    <a class="ez-login__password-visibility-toggler" href="javascript: void(0)">
```

Comment:
Is it an anchor? I think it just `button`

Reply:
Also, I would expect this button will be right after the input not in some separate div together with forgot password